bee-tangle/src/ms_tangle.rs

```diff
@@ -328,6 +328,16 @@ impl<B: StorageBackend> MsTangle<B> {
         self.solid_entry_points.lock().await.insert(sep, index);
     }
 
+    /// Returns a copy of all solid entry points.
+    pub async fn get_solid_entry_points(&self) -> Vec<(SolidEntryPoint, MilestoneIndex)> {
+        self.solid_entry_points
+            .lock()
+            .await
+            .iter()
+            .map(|(k, v)| (*k, *v))
```

Comment:
`.cloned()` or `.copied()` don't work here?

Reply:
yea, they didn't for some reason